Corrected some grammar and spelling errors.

jjm223 opened the pull request:
I've noticed a few errors here and there regarding text in game.

xXKeyleXx commented on MyPet_en.properties:
```diff
@@ -15,51 +15,51 @@ Message.CaptureHelper.NotUserCreated=This Iron Golem <red>wasn't<reset> build by
-Message.Command.PetType.Unknown=<red> Unknown Pettype!
-Message.Command.Release.Dead=<aqua>{0}<reset> is <red>dead<reset> and will never come back. . .
+Message.Command.PetType.Unknown=<red>Unknown pet type!
+Message.Command.Release.Dead=<aqua>{0}<reset> has been released, and is no longer alive...
```
Even if the old one wasn't correct (IDK) it sounded way better. The released part may need some change.

xXKeyleXx commented on MyPet_en.properties:
```diff
@@ -15,51 +15,51 @@ Message.CaptureHelper.NotUserCreated=This Iron Golem <red>wasn't<reset> build by
+Message.LevelSystem.LevelUp=<aqua>{0}<reset> is now level <gold>{1}<reset>.
 Message.LevelSystem.ReachedMaxLevel=<aqua>{0}<reset> has achieved the level cap of <gold>{1}<reset>.
-Message.MultiWorld.NoActivePetInThisWorld=You <red>don't<reset> have a <gold>MyPet<reset> in this world.
+Message.MultiWorld.NoActivePetInThisWorld=You don't have a <gold>MyPet<reset> in this world!
```
Why no red color?

xXKeyleXx commented:
I can't belive that after all these years someone corrected it xD Thanks alot.

jjm223 commented:
Regarding your notes, 
I feel that "is dead and will never come back" doesn't really sound correct, as this is referring to when an animal is released and disappears, not just to when it dies. I agree that my replacement still sounds a little odd though.

I started removing the red color because in my mind it seems to be a bit over the top, and it doesn't really serve much of a purpose. After I was finished I had started to put the red back as I realized it wouldn't be consistent with the other translations, I guess I forgot a line :P

I fixed these two issues. I modified my "released forever" replacement to sound a bit better. I feel that it's a bit more fitting than the original message.

xXKeyleXx commented:
"This branch has conflicts that must be resolved" Could you please do a quick rebase so I can merge it from here?

jjm223 commented:
I took care of it, in case you didn't see.

xXKeyleXx commented:
Thanks :D
